src/components/TPS6208x.stanza

```diff
@@ -4,11 +4,13 @@ defpackage TI-vreg/components/TPS6208x :
   import math
   import jitx
   import jitx/commands
+  import jitx/parts
 
-  import jsl/symbols
-  import jsl/landpatterns
-  import jsl/design/introspection
+  import jsl
+  import jsl/utils
```

Comment:
why do you need `jsl/utils` if you have `jsl`?

Reply:
Because I have a tool in there called `unpack` that didn't make sense to put in JSL. I was worried about name clashes if I were to include it in `jsl`. 

Frankly - it is a stanza language short coming.